### PR TITLE
Update tsd

### PR DESCRIPTION
**Pull request by bpinto**

### CODE REVIEW

**Comment by bpinto** [comment]
As mentioned https://github.com/kefirjs/kefir-test-utils/pull/340#issuecomment-971569800, I'm not sure if these changes are correct since I'm not familiar with typescript.

File: kefir-test-utils.test-d.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ expectType<void>(
     clock.runToLast()
   })
 )
-expectType<Event<string, number>>(logItem({type: 'value', value: 'hello'}, false))
+expectType<Event<string, string>>(logItem({type: 'value', value: 'hello'}, false))
```
Comment (mAAdhaTTah) [inline kefir-test-utils.test-d.ts:22]: Does this fail? I think this is intended to communicate that the Error type can be anything. Which maybe implies it should be `unknown`?

Comment (bpinto) [reply kefir-test-utils.test-d.ts:22]: Yes, it fails. You can see the failures on the [PR](https://github.com/kefirjs/kefir-test-utils/pull/361) created by dependabot.

**Review by mAAdhaTTah — APPROVED** [review]

**Comment by mAAdhaTTah** [comment]
Needs a rebase.

**Comment by bpinto** [comment]
@mAAdhaTTah I've rebased and updated tsd to latest (0.19), it was previous on 0.18.

**Pull request merged**